test/src/unit-var_offsets.cc

```diff
@@ -0,0 +1,607 @@
+/**
```

Comment:
Since we're using the C++ API, let's name this file something like `unit-cppapi-var-offsets`.